packages/perseus/src/widgets/__stories__/radio.stories.jsx

```diff
@@ -8,12 +8,17 @@ import {
     multiChoiceQuestion,
 } from "../__testdata__/radio_testdata.js";
 
-import type {PerseusRenderer} from "../../perseus-types.js";
+import type {
+    PerseusRenderer,
+    PerseusRadioWidgetOptions,
+} from "../../perseus-types.js";
 import type {APIOptions} from "../../types.js";
 
 type StoryArgs = {|
     // Radio Options
     static: boolean,
+    randomize: boolean,
+    randomizeSeed: number,
```

Comment:
So I think I get `randomize`; it seems like a builtin property of the Radio widget for shuffling answers. Glad to see tests added for that.

Could you help my understand the point of `randomizeSeed`? I'm hesitant to add a new property to RendererWithDebugUI because, to me at least, it looks like it's meant to match the API of Renderer (and they should probably be using shared Flow types). If that's the point - to match the API of Renderer to get as close as possible to the production behavior - then I'd be nervous to extend that API just for tests.

Reply:
I wonder if we could separate out the randomization tests and take another run at this?  This PR is quite large and if we separate out the randomization tests we can land this and deal with the specific challenge of randomized radio widget tests separately. 

I see that the radio widget in the `_choiceTransform` checks for `problemNum` and uses that as the random seed. If it's undefined though, we delegate to the `Util.random()` function to get the seed. I wonder if we could stop passing in the `problemNum` in the debug renderer and always mock `Util.random()` so that we have a constant seed always.  I suspect we _might_ want to change the seed in some tests, but we could cross that bridge if/when we get there.

Reply:
@handeyeco Hmm, I didn't realize RenderWithDebugUI was trying to match Renderer, so that totally makes sense. Like @jeremywiebe mentioned, the point of randomizeSeed is to allow us to change the seed, specifically inside of Storybook. I wanted us/testers to see that randomize does work and that the problemNum (aka randomizeSeed) does actually affect the order of the answers. I partially wanted to explore this because of the [ticket](https://khanacademy.atlassian.net/browse/LP-11571) out regarding randomize not working in certain situations, so I wanted to have a strong basis to allow for testing. It's low priority though, so definitely something that can be explored later.

ProblemNum was constant before I added Randomize Seed. It was just set to 0 by default. I will just switch it back here while separating out the randomization tests. Thank you both for the feedback!

Reply:
Just pulled out all the randomize related content. :)